inventory/byo/hosts.example

```diff
@@ -78,5 +78,5 @@ ose3-etcd[1:3]-ansible.test.example.com
 
 # host group for nodes
 [nodes]
-ose3-master[1:3]-ansible.test.example.com openshift_scheduleable=False
+ose3-master[1:3]-ansible.test.example.com openshift_schedulable=False
```

Comment:
This naming change will also need to be updated in the references in openshift-docs as well and openshift-extras (enterprise installer wrapper). We might also want to handle both spellings for a transition period as well.